image_layers/query-map-image-sublayer/src/main/java/com/esri/samples/query_map_image_sublayer/QueryMapImageSublayerSample.java

```diff
@@ -72,8 +73,12 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with a basemap and set its initial viewpoint
-      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style and set its initial viewpoint
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
       Point initialLocation = new Point(-12716000.00, 4170400.00, SpatialReferences.getWebMercator());
       Viewpoint viewpoint = new Viewpoint(initialLocation, 6000000);
```

Comment:
I know you haven't touched this code, but might be worth updating this?
I think it makes more sense to set the SR on the map, rather than on an individual point, and then not specify a SR when creating points/viewpoints and adding them to our map. 
Then we ould also get rid of the `initialLocation` and create the `Viewpoint` directly with the coordinates.

Reply:
Thanks Jon your comments have helped me untangle that. Have done some tweaks and tested.

Reply:
Nice work, it looks so much nicer 🚀